### PR TITLE
Document CryptoCore getting started

### PR DESCRIPTION
**Pull request by JakeSCahill**
Needs initial content

### CODE REVIEW

File: iot/0.1/cryptocore/introduction/get-started.md [hunk 1]
```diff
@@ -1,50 +1,111 @@
...
+
+1. Open a serial terminal
+
+  ```bash
+  cd ~/iccfpga-utils
+  ./start_serial.sh
+  ```
+
+2. Use the API to request the current version of the CryptoCore program
+
```
[the remark this answers is not in the document]
Comment (l3wi) [reply iot/0.1/cryptocore/introduction/get-started.md]: A locahost CURL example would be great here. 

```
curl -XPOST -H 'X-IOTA-API: 1' -H "Content-type: application/json" -d '{"command":"version"}' 'localhost:8537'
```

Made this with https://curlbuilder.com/

File: iot/0.1/cryptocore/introduction/get-started.md [hunk 2]
```diff
@@ -1,50 +1,111 @@
...
+After setting up the CryptoCore with all the necessary software, you can test it by running the 'hello world' program.
+
+1. Open a serial terminal
+
+  ```bash
+  cd ~/iccfpga-utils
+  ./start_serial.sh
+  ```
+
```
Comment (l3wi) [inline iot/0.1/cryptocore/introduction/get-started.md]: Maybe add a comment about what this has done. eg. Downloaded the CC program, flashed it, installed the local driver, started the server.

File: iot/0.1/cryptocore/introduction/get-started.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (JakeSCahill) [reply iot/0.1/cryptocore/introduction/get-started.md]: This example is for using the API over a serial connection.

When the REST server is added to the script, we can add this example 😄

File: iot/0.1/cryptocore/introduction/get-started.md (hunk 2 above)
Comment (JakeSCahill) [reply iot/0.1/cryptocore/introduction/get-started.md]: This step just opens a serial terminal

File: iot/0.1/cryptocore/introduction/get-started.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+**The CryptoCore is IOTA hardware designed for applications that need fast, dedicated proof of work and a secure memory. The device consists of an IOTA CryptoCore FPGA (ICCFPGA) module and a development board that doubles as a Raspberry Pi HAT, making it perfect for stand-alone applications and/or quick prototyping. In this guide, you learn how to get started with the CryptoCore on the Raspberry Pi 3/4.**
+
+![CryptoCore](../images/cryptocore.jpg)
+
+## Hardware
+
+To complete this guide, you need the following:
+- A PC
+- A Raspberry Pi 3/4 with Raspbian installed on a micro SD card, SSH enabled, and the serial interface enabled
+- A CryptoCore module
```
Comment (Citrullin) [inline iot/0.1/cryptocore/introduction/get-started.md]: Cryptocore Modul and RPi HAT. The RPi HAT is missing

File: iot/0.1/cryptocore/introduction/get-started.md [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+
+On the CryptoCore development board, make sure that jumpers J7, J8, and J9 are set to `PI`:
```
Comment (Citrullin) [inline iot/0.1/cryptocore/introduction/get-started.md]: Can we use one naming convention? Development board or HAT?

File: iot/0.1/cryptocore/references/scripts.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
+# Utility scripts
+
+**To make it easier to use the CryptoCore with a Raspberry Pi, we have the following scripts that take care of some common tasks.**
```
Comment (Citrullin) [inline iot/0.1/cryptocore/references/scripts.md]: Crypto Core HAT, development baord or whatever. The module itself is a bit hard to use with the RPi alone

File: iot/0.1/cryptocore/introduction/get-started.md (hunk 4 above)
Comment (JakeSCahill) [reply iot/0.1/cryptocore/introduction/get-started.md]: We use development board in the manual. Maybe we should use that?

File: iot/0.1/cryptocore/introduction/get-started.md (hunk 3 above)
Comment (JakeSCahill) [reply iot/0.1/cryptocore/introduction/get-started.md]: Shall we say CryptoCore development kit?

**Pull request merged**